pkg/config/serving/server.go

```diff
@@ -58,7 +58,10 @@ func ToServerConfig(ctx context.Context, servingInfo configv1.HTTPServingInfo, a
 	}
 
 	if !authorizationConfig.Disabled {
-		authorizationOptions := genericapiserveroptions.NewDelegatingAuthorizationOptions()
+		authorizationOptions := genericapiserveroptions.NewDelegatingAuthorizationOptions().
+			WithAlwaysAllowPaths("/healthz", "/healthz/", "/readyz"). // this allows the kubelet to always get health and readiness without causing an access check
+			WithAlwaysAllowGroups("system:masters") // in a kube cluster, system:masters can take any action, so there is no need to ask for an authz check
```

Comment:
is this for the `metrics` endpoint ?

Reply:
I was asking because from what I have seen this method is used in the operators to expose the `metrics` endpoint.